pkgs/top-level/all-packages.nix

```diff
@@ -5555,6 +5555,7 @@ with pkgs;
   cudaPackages_11_5 = callPackage ./cuda-packages.nix { cudaVersion = "11.5"; };
   cudaPackages_11_6 = callPackage ./cuda-packages.nix { cudaVersion = "11.6"; };
   cudaPackages_11_7 = callPackage ./cuda-packages.nix { cudaVersion = "11.7"; };
+  cudaPackages_11_8 = callPackage ./cuda-packages.nix { cudaVersion = "11.8"; };
   cudaPackages_11 = cudaPackages_11_7;
```

Comment:
Don't we want `cudaPackages_11` to be an alias for the latest available version?

Reply:
Yes, but as the current version of cudnn is not compatible with version 11.8 of cuda, I've not updated the alias.

If there is a consensus, no problem to update it

Reply:
Ah, then yes, keep it back for now.

Reply:
Latest cudnn is 8.6.0, compatible with cuda 11.8

Reply:
(I'd do the bump in a separate commit, because everything else we can backport to 22.05)

Reply:
> Latest cudnn is 8.6.0, compatible with cuda 11.8

Thx, I'd missed that. Push an update in this PR.